browser/ui/color/brave_color_mixer.cc

```diff
@@ -388,13 +337,79 @@ void AddBraveDarkThemeColorMixer(ui::ColorProvider* provider,
   mixer[kColorFeaturePromoBubbleForeground] = {SkColorSetRGB(0xC6, 0xC8, 0xD0)};
   mixer[kColorFeaturePromoBubbleCloseButtonInkDrop] = {
       GetToolbarInkDropColor(mixer)};
+}
+
+}  // namespace
+
+SkColor GetLocationBarBackground(bool dark, bool priv, bool hover) {
+  if (priv) {
+    return hover ? color_utils::HSLShift(kPrivateLocationBarBgBase,
+                                         {-1, -1, 0.54})
+                 : kPrivateLocationBarBgBase;
+  }
+
+  if (dark) {
+    return hover ? kDarkLocationBarHoverBg : kDarkLocationBarBgBase;
+  }
+
+  return hover ? color_utils::AlphaBlend(SK_ColorWHITE,
+                                         SkColorSetRGB(0xf3, 0xf3, 0xf3), 0.7f)
+               : SK_ColorWHITE;
+}
+
+// Omnibox result bg colors
+SkColor GetOmniboxResultBackground(int id, bool dark, bool priv) {
+#if !defined(USE_AURA)
+  ui::NativeTheme* native_theme = nullptr;
+#else
+  // For high contrast, selected rows use inverted colors to stand out more.
+  ui::NativeTheme* native_theme = ui::NativeTheme::GetInstanceForNativeUi();
+#endif  // !defined(USE_AURA)
+  bool high_contrast =
+      native_theme && native_theme->UserHasContrastPreference();
+  float omnibox_opacity = 0.0f;
+  if (id == kColorOmniboxResultsBackgroundHovered) {
+    omnibox_opacity = kOmniboxOpacityHovered;
+  } else if (id == kColorOmniboxResultsBackgroundSelected) {
+    omnibox_opacity = kOmniboxOpacitySelected;
+  }
+
+  SkColor color;
+  if (priv) {
+    color = high_contrast ? color_utils::HSLShift(kPrivateLocationBarBgBase,
+                                                  {-1, -1, 0.45})
+                          : kPrivateLocationBarBgBase;
+  } else if (dark) {
+    color = high_contrast ? gfx::kGoogleGrey900 : kDarkLocationBarBgBase;
+  } else {
+    color = SK_ColorWHITE;
+  }
+  return color_utils::BlendTowardMaxContrast(
+      color, base::ClampRound(omnibox_opacity * 0xff));
+}
+
+void AddBravifiedChromeThemeColorMixer(
+    ui::ColorProvider* provider,
+    const ui::ColorProviderManager::Key& key) {
+  if (key.custom_theme)
```

Comment:
So, this is true when custom theme is installed on normal window or it's private window right? but the latter won't call this, as it always wants dark theme.

Reply:
Correct.